### PR TITLE
feat: Update TSC Banner

### LINKED ISSUE #505: [Advanced]: Update TSC page banner image
### 🧑‍🔬 Advanced Issue

Welcome! This is an **[Advanced Issue](https://github.com/issues?q=is%3Aopen%20is%3Aissue%20org%3Ahiero-ledger%20archived%3Afalse%20no%3Aassignee%20label%3A%22advanced%22%20repo%3Ahiero-ledger%2Fhiero-website)** touching core website architecture.

It is designed for expert contributors who have demonstrated deep architectural understanding and a proven track record of high-quality contributions.

### 🐞 Problem Description

Currently our TSC page banner image does not follow our desired design. 

With the new vote of the website redesign we would like to update this to reflect the design of our Homepage. 

### 🛠️ Implementation Notes

Implement the same feature on the homepage into the TSC banner section. 

The design should match the figma file speciation outlined here
https://www.figma.com/design/hZztcL7ukzpYLlIklBFzwP/Hiero-Website-Design-Daniel?node-id=1-3&t=3UYJ7D1bhd7NWafv-0

PUSH WORK TO websiteRedesign BRANCH

Markdown Specification: 
https://github.com/hiero-ledger/hiero-website/blob/main/docs/design-specs.md

**Blog & Content Workflow:**
- [Writing Blog Posts](https://github.com/hiero-ledger/hiero-website/blob/main/docs/blogs.md) — covers the current posting flow, front matter format, and content conventions

**Project Conventions:**
- Components live in `src/components/` — follow existing patterns for props, naming, and file structure
- Pages are under `src/app/` using the Next.js App Router
- Content (blog posts, etc.) lives in `content/`
- Styling uses [Tailwind CSS](https://tailwindcss.com/docs)

### 🧠 Advanced Contributors — Prerequisites & Expectations

> [!CAUTION]
> **Advanced issues are the highest-risk work in this project. We will reject PRs that do not meet these standards.**

### 🏁 Concrete Prerequisites
- **Advanced Web Development:** Proficient with React, Next.js, and TypeScript.
- **Expertise:** Deep architectural understanding of the website's component structure, routing, and build pipeline.
- **Proven History:** Successfully completed **≥ 10 intermediate issues** in this repo.
- **Consistency:** **≥ 3–4 months** of active, human-led contributions to this project.

> [!NOTE]
> **CI/CD Exception:** For issues focused on **GitHub Actions / Workflows**, the repo-specific thresholds above may be waived if the contributor demonstrates advanced-level proficiency in CI/CD.

### ⚠️ AI Usage Policy

- Using AI to generate code for Advanced issues is **strictly discouraged**
- AI may be used to help explain file relationships, but cannot be the main source of research.
- Submitting AI-generated or unvalidated code is grounds for **immediate closure**

### ⏱️ Timeline & Workflow
- **Typical time:** ~1 month / ~50 hours.
- 🔴 Completing an advanced issue in 1–3 days is a **red flag** and will likely be rejected.
- **Suggested:** Post your proposed architectural approach as a comment and wait for explicit maintainer approval **before writing any code.**

### 🔬 Technical Domains

- [ ] **Advanced React / Next.js** (Server Components, App Router, dynamic routing, middleware)
- [ ] **Component Architecture** (design systems, reusable patterns, composition)
- [ ] **Performance & Optimization** (Core Web Vitals, image optimization, code splitting, caching)
- [ ] **Accessibility** (WCAG compliance, screen reader support, keyboard navigation)
- [ ] **Build & Deployment** (Next.js config, CI/CD, hosting/deployment integration)
- [ ] **Testing** (unit, integration, E2E, accessibility testing, visual regression)

### 🛡️ Quality & Review Standards

Advanced PRs must be **"safe, maintainable, architecturally sound, and production-ready."**

1. **Architectural Fit:** The solution must fit naturally into the existing website architecture.
2. **Security & Correctness:** Evaluate all logic for injection risks, XSS vulnerabilities, or state corruption issues. 
3. **Maintainability:** Code must be short and clear enough for any other maintainer to debug without your assistance. 
4. **Backward Compatibility:** Existing pages, routes, and components must not break. If a breaking change is required, it must be explicitly managed.
5. **Comprehensive Testing:** Must include unit and integration tests covering all new logic paths, edge cases, and failure modes. AI-generated tests based on AI-generated code are grounds for immediate rejection.

### ✅ PR Quality Checklist

Before opening your PR, the contributor must confirm:
- [ ] I have spent the majority of my time researching the problem and solution space extensively, including reviewing relevant code, documentation, and external resources.
- [ ] I understand the system-wide impact of these changes on affected modules and performance.
- [ ] The design fits with the current website's architectural approaches.
- [ ] I have tested my changes extensively across browsers and screen sizes.
- [ ] Every line of code is personally understood and explainable.

### 📚 Resources & Support

**References:**
- [Next.js Documentation](https://nextjs.org/docs)
- [React Documentation](https://react.dev)
- [Tailwind CSS Documentation](https://tailwindcss.com/docs)
- [Vitest Documentation](https://vitest.dev)
- [Writing Blog Posts](https://github.com/hiero-ledger/hiero-website/blob/main/docs/blogs.md)

**🆘 Stuck?**
- [Community Calls](https://zoom-lfx.platform.linuxfoundation.org/meetings/hiero?view=week)
- [Discord](https://github.com/hiero-ledger/sdk-collaboration-hub/blob/main/guides/issue-progression/for-developers/discord.md)

### PR DESCRIPTION
**Pull request by aceppaluni**
# Description

This PR update the TSC banner image for the website redesign

## Changes Made
- [x] Added... Updated banner 
- [ ] Modified...
- [ ] Fixed...

## Related Issues

Fixes: #505 

## Checklist
- [ ] Tests added/updated
- [ ] Documentation updated
- [ ] Linting passes
- [x] Branch up-to-date with main

## Summary by CodeRabbit

* **New Features**
  * Added an animated network visual with connected nodes and person icons to the TSC hero section.
  * Redesigned the hero section with a dark burgundy gradient, governance badge, divider, larger heading, and updated messaging.
  * Retained quick access to the charter and calendar.

* **Style**
  * Improved visual consistency across the About, Committee Members, and Governance Resources sections.

### CODE REVIEW

File: src/app/tsc/page.tsx [hunk 1]
```diff
@@ -32,6 +32,394 @@ const governanceResources = [
...
+
+    const nodes: NetworkNode[] = [
+      {
+        x: 0.04,
+        y: 0.25,
+        radius: 5,
+        speedX: 0.0005,
+        speedY: 0.0004,
+        phase: 0.5,
+        person: false,
```
Comment (exploreriii) [inline src/app/tsc/page.tsx:83]: The animation looks really good

I notice the TSC animation has both people and non people nodes 

Currently the people nodes are connecting with non people nodes 

I wonder if there is any benefit for the tsc animation to instead have connections between tsc nodes, and each of these tsc people can then have connections to nodes ? 

So more like this would look like a central tsc cluster, similar to how they as a team make a coordinated decision

File: src/app/tsc/page.tsx [hunk 2]
```diff
@@ -32,6 +32,394 @@ const governanceResources = [
...
+      x: number,
+      y: number,
+      pulse: number,
+    ) => {
+      if (node.person) {
+        context.save();
+
+        // Outer network rings
+        context.strokeStyle = "rgba(255,255,255,0.10)";
+        context.lineWidth = 1;
```
Comment (exploreriii) [inline src/app/tsc/page.tsx:277]: Maybe we should put settings like at the top of the file then pass them down in the relevant multiple places,9 so if we ever need to edit the colours we can do it faster with more safety

**Review by exploreriii — APPROVED** [review]

**Review by exploreriii — CHANGES_REQUESTED** [review]
needs to be made to web dev branch

**Review by exploreriii — APPROVED** [review]

**Review by danielmarv — APPROVED** [review]
